src/app/AlwaysOn.HealthService/README.md

```diff
@@ -6,11 +6,15 @@ The HealthService is an app component that is running along other components lik
 
 The idea is, first of all, if the cluster itself is down, the health service won't respond at all. When the service is up and running, it performs periodic checks against various components of the solution:
 
-- It attempts to do a simple query against Cosmos DB
+- It attempts to do a simple query against Cosmos DB (read and write)
 - It attempts to send a message to Event Hub (the message will be filtered out by the background worker)
 - It looks up a state file on the storage account. This file can be used to turn off a region, even while the other checks are still working ok.
 
-All health check results are cached in memory for a configurable number of seconds (by default 10) so that not every call to the API results in backend calls. While this does add a small potential latency in detecting outages, it also reduces the additional cluster load generated by health checks.
+In addition to the direct "pings" to the downstream services, the HealthService also queries Azure Monitor for the current HealthScore (as defined in the Health Model).
+While there is an ingestion latency with Azure Monitor (typically a few minutes), the HealthScore does also take into account additional signals which are not covered by the pings of the HealthService.
+Hence, the combination of both enables the HealthService to do both, a real-time picture of the most critical dependencies of the API, as well as a holistic view of the entire stamp as defined in the health model.
```

Comment:
If these are supposed to be another paragraphs, it's missing space.

Reply:
fixed